src/features/unified-share-modal/utils/convertData.js

```diff
@@ -180,3 +184,54 @@ export const convertSharedLinkPermissions = (newSharedLinkPermissionLevel: strin
     });
     return sharedLinkPermissions;
 };
+
+/**
+ * Convert a response from the Item Collaborations API into the object that the USM expects.
+ * @param {Collaborations} collabsAPIData
+ * @param {string | null | undefined} ownerEmail
+ * @param {boolean} isCurrentUserOwner
+ */
+export const convertCollabsResponse = (
+    collabsAPIData: Collaborations,
+    ownerEmail: ?string,
+    isCurrentUserOwner: boolean,
+): collaboratorsListType => {
+    let collaborators = [];
+
+    const { entries } = collabsAPIData;
+
+    if (entries.length) {
+        const ownerEmailDomain = ownerEmail ? ownerEmail.split('@')[1] : null;
```

Comment:
Is there any chance that the email is not well formed? i.e. doesn't have `@`

Reply:
Should be fine, since the emails are saved directly from API responses. I added a regex check to this line, just in case.